### ISSUE TITLE
db Table for holding annotations. 

### ISSUE THREAD

**Issue by tbags** [tbags/django-image-annotation-tool#1]
Create a table structure and corresponding sql scheme for holding all the necessary annotations.

Start from running the server download a custom annotations JSON objects.

**Comment by tbags** [comment]
http://dbdesigner.net

**Comment by tbags** [comment]
Any updates?

**Comment by prateethvnayak** [comment]
@tbags Yes. Have the draft ready based on last meeting. Will open a google team drive
